wp-content/themes/core/assets/pcss/media-queries/_variables.pcss

```diff
@@ -18,3 +18,13 @@
 @custom-media --viewport-large (min-width: 1200px);
 @custom-media --viewport-xlarge (min-width: 1260px);
 @custom-media --viewport-retina (-webkit-min-device-pixel-ratio: 2), (min-resolution: 192dpi), (min-resolution: 2dppx);
+
+/* --------------------------------------------------
+ * WP Breakpoints
+ * https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_breakpoints.scss
```

Comment:
This was immensely helpful when building out LendNation to reference while styling patterns and follow WP's breakpoints for changes on blocks like Media & Text, Columns, etc.

Doesn't seem like WP has a standardized set so I've opted to add the ones we used most often.